python/insights_ansible_playbook_lib/__init__.py

```diff
@@ -204,3 +204,13 @@ def get_revocation_digests(playbook: str, gpg_key: bytes) -> set[bytes]:
     revoked: list[dict] = play.get("revoked_playbooks", [])
     digests = set(bytes(bytearray.fromhex(item["hash"])) for item in revoked)
     return digests
+
+
+def represent_none(self: yaml.Dumper, data: None) -> yaml.ScalarNode:
+    """Represent null as an empty string.
+
+    :param self: The YAML object.
+    :param data: The data to be represented.
+    :returns: An empty string.
+    """
+    return self.represent_scalar("tag:yaml.org,2002:null", "")
```

Comment:
It seems this would better fit to the `..._lib.serialization` package, since it manages YAML stuff. wdyt?